netbox/dcim/forms/filtersets.py

```diff
@@ -908,7 +908,7 @@ class VirtualChassisFilterForm(TenancyFilterForm, NetBoxModelFilterSetForm):
 class CableFilterForm(TenancyFilterForm, NetBoxModelFilterSetForm):
     model = Cable
     fieldsets = (
-        (None, ('q', 'filter_id', 'tag')),
+        (None, ('q', 'filter_id', 'unterminated', 'tag')),
```

Comment:
The `unterminated` field should go under one of the model-specific sections below.